test/get_previous_releases.py

```diff
@@ -148,10 +148,38 @@ def download_binary(tag, args) -> int:
     ret = subprocess.run(['tar', '-zxf', tarball, '-C', tag,
                           '--strip-components=1',
                           'bitcoin-{tag}'.format(tag=tag[1:])]).returncode
-    if ret:
+    if ret != 0:
+        print("Failed to extract the {tag} tarball")
```

Comment:
This change looks good, but I'd prefer to see it its own commit, to keep every commit focused.